airbyte-integrations/connectors/source-intercom/source_intercom/source.py

```diff
@@ -300,6 +300,14 @@ class ConversationParts(ChildStreamMixin, IncrementalIntercomStream):
 
     def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
         return f"/conversations/{stream_slice['id']}"
+    
+    def parse_response(self, response: requests.Response, stream_state: Mapping[str, Any], **kwargs) -> Iterable[Mapping]:
+        records = super().parse_response(response, stream_state, **kwargs)
+        conversation_id = response.json().get('id')
+        for conversation_part in records:
+            if 'conversation_id' not in conversation_part:
+                conversation_part['conversation_id'] = conversation_id
```

Comment:
```suggestion
           conversation_part.setdefault("conversation_id", "conversation_id")
```

Reply:
Done in [c88184b](https://github.com/airbytehq/airbyte/pull/11206/commits/c88184b939d605be30fe5d7c3e0213bdfb3a48b9).